jobs/e2e-runner.groovy

```diff
@@ -0,0 +1,105 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+import utilities.StatusUpdater
+
+[ [type: 'master', branch: 'master'],
+  [type: 'pr', branch: '${sha1}'],
+].each { Map config ->
+  isMaster = config.type == 'master'
+  isPR = config.type == 'pr'
+
+  name = isMaster ? "e2e-runner-master" : "e2e-runner-pr"
```

Comment:
or actually I think our precedent was master job = `"e2e-runner"` and pr job = `"e2e-runner-pr"`... if thats the case and we want to stick w/ convention, we'd keep the same logic, minus the `"master"` in the master job name